ansible/y_cable_simulator_injector.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: generate y_cable_simulator driver for {{ dut_name }}
+  template: src=dualtor/y_cable_simulator_client.j2
+            dest=/tmp/y_cable_simulator_client.py
+  delegate_to: "{{ dut_name }}"
+
+- name: inject y_cable_simulator to pmon container for {{ dut_name }}
+  shell: docker cp /tmp/y_cable_simulator_client.py pmon:/usr/lib/python3/dist-packages
```

Comment:
@vdahiya12 Is this the appropriate place to put the y_cable_simulator_client in the pmon docker?